modules/microvm/virtualization/microvm/appvm.nix

```diff
@@ -106,6 +109,19 @@ let
                 lib.mkIf configHost.ghaf.virtualization.microvm.idsvm.mitmproxy.enable
                   [ ./idsvm/mitmproxy/mitmproxy-ca/mitmproxy-ca-cert.pem ];
 
+              security.rtkit.enable = vm.ghafAudio;
+              users.extraUsers.ghaf.extraGroups =
+                lib.optionals vm.ghafAudio ["audio" "video"];
+
+              hardware.pulseaudio = lib.mkIf vm.ghafAudio {
+                enable = true;
+                extraConfig = ''
+                    load-module module-tunnel-sink-new sink_name=${vm.name}.speaker server=audio-vm:4713 reconnect_interval_ms=1000
+                    load-module module-tunnel-source-new source_name=${vm.name}.mic server=audio-vm:4713 reconnect_interval_ms=1000
+                '';
+                package = pkgs.pulseaudio-ghaf;
+              };
+
```

Comment:
Would it make sense to move this to a separate module, like common/vm-audio.nix, and import it here, similar to how we have vm-networking and storage?

Reply:
added ghaf-audio module

Reply:
Why not move the definition of the ghafAudio option to the module as well and use lib.mkIf cfg.enable? This way it will be consistent with other modules and easier to read. You can take a look at storagevm.nix as an example or pretty much any other module in Ghaf.

Reply:
Added ghaf-audio as a separate module with an option to turn it on/off.